README.org

```diff
@@ -18,3 +18,6 @@ Here's a table describing the bindings
 | =ediff-suspend=             | =z=              | =C-z=          |
 | scroll left                 | =>=              | =zh=           |
 | scroll right                | =<=              | =zl=           |
+| copy A region to B's region | a                | l              |
+| copy B region to A's region | b                | h              |
```

Comment:
`l` and `h` don't seem to appear in the changes below. 

I also am not sure they're great choices. What happens when you're diffing 3 files?